uk.co.compucorp.civicrm.tasksassignments/js/test/controllers/modal/modal-document.controller.spec.js

```diff
@@ -612,6 +614,31 @@ define([
       });
     });
 
+    describe('when user click on the document wrench icon', function () {
+      var onPopupFormSuccess;
```

Comment:
`var onPopupFormSuccess, url;`
you can merge vars together if there is no assignment